pkg/controller/servicebindingrequest/planner.go

```diff
@@ -44,15 +45,37 @@ func (p *Planner) searchCR(kind string) (*unstructured.Unstructured, error) {
 	logger.Info("Searching for CR instance...")
 
 	cr, err := p.client.Resource(gvr).Namespace(p.sbr.GetNamespace()).Get(bss.ResourceRef, opts)
+
 	if err != nil {
-		logger.Error(err, "during reading CR")
+		logger.Info("during reading CR")
 		return nil, err
 	}
 
 	logger.WithValues("CR.Name", cr.GetName()).Info("Found target CR!")
 	return cr, nil
 }
 
+// searchCRD based on a CustomResourceDefinitionDescription and name, search for the object.
```

Comment:
This comment needs to be updated. It's the same as the comment for `searchCR()` method